plugins/optimization-detective/class-od-url-metric.php

```diff
@@ -46,6 +46,7 @@ final class OD_URL_Metric implements JsonSerializable {
 	 * Constructor.
 	 *
 	 * @param array $data URL metric data.
+	 * @phpstan-param Data $data URL metric data.
 	 *
```

Comment:
Lets separate custom params from other params. See [this example](https://github.com/WordPress/plugin-check/blob/trunk/includes/Checker/Check_Collection.php#L40-L51).

```suggestion
	 *
	 * @phpstan-param Data $data URL metric data.
	 *
	 * @param array $data URL metric data.
	 *
```